src/ui/components/TransferGrain.js

```diff
@@ -51,69 +106,90 @@ export const TransferGrain = ({ledger, setLedger}: Props) => {
   };
 
   return (
-    <div
-      style={{
-        width: "80%",
-        margin: "0 auto",
-        background: "white",
-        padding: "0 5em 5em",
-      }}
-    >
-      <h1>Transfer Grain</h1>
-      <form onSubmit={(e) => submitTransfer(e)}>
-        <label>From</label>
-        <AccountDropdown ledger={ledger} setCurrentIdentity={setSender} />
-        <br />
-        <label>To</label>
-        <AccountDropdown ledger={ledger} setCurrentIdentity={setReceiver} />
-        <p>
-          <label htmlFor="amount">Amount</label> <br />
-          <input
-            type="number"
-            name="amount"
-            min="0"
-            step="any"
-            placeholder={`max: ${maxAmount}`}
-            required
-            value={amount}
-            onChange={(e) => setAmount(e.currentTarget.value)}
-          />
-          <span>{sourceIdentity && ` max: ${format(maxAmount, 5)}`}</span>
-        </p>
-        <p>
-          <label htmlFor="memo">Memo</label> <br />
-          <input
+    <form onSubmit={(e) => submitTransfer(e)}>
+      <div className={classes.root}>
+        <div className={classes.centerRow}>
+          <h1>Transfer Grain</h1>
+        </div>
+        <div className={classes.centerRow}>
+          <div
+            className={`${classes.dropdown} ${classes.centerRow} ${classes.third}`}
+          >
+            <AccountDropdown
+              ledger={ledger}
+              setCurrentIdentity={setSender}
+              placeholder="From..."
+            />
```

Comment:
I like the UX of rendering the balance in this component, but after a transfer occurs, we're still left with the balance prior to the transfer. We'll need to utilize useEffect or ideally just utilize props directly to render out the balance so that the render isn't one step behind the actual state. I had to troubleshoot a similar issue where the component state is one render behind the latest prop value and this was my solution:
https://github.com/sourcecred/sourcecred/blob/74b4df4ab628968a5b88b81b1d18fc19c919dc03/src/ui/components/AliasSelector.js#L62-L64

This looks like a bug that snug in through #2079